tests/e2e/scheduler/scheduler_test.go

```diff
@@ -168,11 +168,13 @@ func TestJobs(t *testing.T) {
 			require.NoError(t, err)
 
 			// Check if the length of triggeredJobs matches the expected length of scheduled jobs
-			assert.Equal(c, numIterations*numJobsPerGoRoutine, len(triggeredJobs))
+			assert.Len(c, triggeredJobs, numIterations*numJobsPerGoRoutine)
 		}, 10*time.Second, 100*time.Millisecond)
 		t.Log("Done.")
 	})
 
+	time.Sleep(time.Second * 5)
```

Comment:
Why add a `time.sleep`? If the tests are interfering with one another can we instead add a `ttl` to the individual jobs or add a`delete` call at the end of the first `t.run`?

Reply:
Thanks, was a left over from my debugging. I agree, I've removed it